app/routes/missions.einstein-probe/route.mdx

```diff
@@ -33,6 +33,13 @@ The [Einstein Probe (EP)](https://ep.bao.ac.cn/ep/) is a mission of the Chinese
 
 **JSON-Serialized GCN Notices Types in GCN Kafka:**
 
+Einstein Probe distributes alerts and localizations detections of gamma-ray transients. These notices are published on the GCN Kafka topic `gcn.notices.einstein_probe.wxt.alert`.
+
+The [GCN schema](/schema/stable/gcn/notices/einstein_probe/wxt/alert.schema.json) and
+example [JSON message](/schema/stable/gcn/notices/einstein_probe/wxt/alert.schema.example.json) files are available to use. See the [Schema Browser](/docs/schema/stable/gcn/notices/einstein_probe/wxt/alert.schema.json) for more information on the properties defined in the schema.
```

Comment:
```suggestion
example [JSON message](/schema/stable/gcn/notices/einstein_probe/wxt/alert.schema.example.json) files are available to use for [Einstein Probe Schema](/schema/stable/gcn/notices/einstein_probe). See the [Schema Browser](/docs/schema/stable/gcn/notices/einstein_probe/wxt/alert.schema.json) for more information on the properties defined in the schema.
```

Reply:
@Vidushi-GitHub, why are you suggesting changes on your own PR? If you want to change it, just make the change.